source/trainer.py

```diff
@@ -0,0 +1,147 @@
+# RUN 3 lines below in a seperate cell in Google Colab
+# !pip install transformers tokenizers wandb huggingface_hub
+# from huggingface_hub import notebook_login
+# notebook_login()
+
+import os
+from pathlib import Path
+from transformers import (
+    PreTrainedTokenizerFast,
+    DataCollatorForLanguageModeling,
+    GPT2Config,
+    GPT2LMHeadModel,
+    TrainingArguments,
+    Trainer,
+)
+from tokenizers import Tokenizer
+from tokenizers.models import WordLevel
+from tokenizers.pre_tokenizers import WhitespaceSplit
+from tokenizers.trainers import WordLevelTrainer
+from datetime import datetime
+import wandb
+
+# CONFIG:
+TRAIN_FROM_CHECKPOINT = None  # Example: checkpoint-80000
+EVAL_STEPS = 1000
```

Comment:
if we have >300_000 steps I would suggest to set this much higher... 
maybe EVAL_STEPS could be by default at least 2 % of the total amount of steps?